plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeModule.java

```diff
@@ -152,14 +148,6 @@ public void setup(Binder binder)
         tableProcedures.addBinding().toProvider(OptimizeTableProcedure.class).in(Scopes.SINGLETON);
     }
 
-    @Singleton
```

Comment:
This removal appears also in https://github.com/trinodb/trino/pull/17092/commits/626039f75bfc40b61538845610936cb0fe7973bc

Reply:
that's a similar binding, but not same?